wp-web-push/wp-web-push-main.php

```diff
@@ -106,6 +111,52 @@ public static function on_transition_post_status($new_status, $old_status, $post
 
     update_option('webpush_notification_count', $notification_count);
   }
+
+  public static function get_trigger_by_key_value($key, $value) {
+    foreach(self::$ALLOWED_TRIGGERS as $trigger) {
+      if($trigger[$key] === $value) {
+        return $trigger;
+      }
+    }
+    return false;
+  }
+
+  public static function get_triggers_by_key_value($key, $value) {
+    $matches = array();
+
+    foreach(self::$ALLOWED_TRIGGERS as $trigger) {
+      if($trigger[$key] === $value) {
+        $matches[] = $trigger;
+      }
+    }
+    return $matches;
+  }
+
+  private static function add_trigger_handlers() {
+    $enabled_triggers = get_option('webpush_triggers');
+    foreach($enabled_triggers as $trigger) {
+        $trigger_detail = self::get_trigger_by_key_value('key', $trigger);
+
+        if($trigger_detail) {
+          echo $trigger_detail['action'];
+          //add_action($trigger_detail['hook'], array(self, $trigger_detail['action']), 10, 3);
+          add_action($trigger_detail['hook'], array($this, $trigger_detail['action']), 10, 3);
```

Comment:
Also try setting WP_DEBUG to true in wp-config.php, maybe it'll show you something.